geode-dunit/src/distributedTest/java/org/apache/geode/test/dunit/tests/BasicDistributedTest.java

```diff
@@ -154,8 +154,9 @@ public void testRemoteInvokeAsync() throws Exception {
     String name = getUniqueName();
     String value = "Hello";
 
-    AsyncInvocation async1 = vm0.invokeAsync(() -> remoteBind(name, value)).join().checkException();
-    AsyncInvocation async2 =
+    AsyncInvocation<?> async1 =
+        vm0.invokeAsync(() -> remoteBind(name, value)).join().checkException();
+    AsyncInvocation<?> async2 =
```

Comment:
This can be:
```
    AsyncInvocation<Void> async1 =
        vm0.<Void>invokeAsync(() -> remoteBind(name, value)).join().checkException();
    AsyncInvocation<Void> async2 =
        vm0.<Void>invokeAsync(() -> remoteValidateBind(name, value)).join().checkException();
```

Reply:
Done.